tests/shell/src/main.c

```diff
@@ -34,8 +34,8 @@ static void test_shell_execute_cmd(const char *cmd, int result)
 static void test_cmd_help(void)
 {
 	test_shell_execute_cmd("help", 0);
-	test_shell_execute_cmd("help -h", 0);
-	test_shell_execute_cmd("help --help", 0);
+	test_shell_execute_cmd("help -h", 1);
```

Comment:
We do not expect an error here. We expect a feedback that command has been not executed.

- +1: shell printed help but command not executed
- 0: command executed sucssfully
- -ERROR_CODE: command not executed because of an error or an error occured during execution.